changelog/pattern-deprecate.dd

```diff
@@ -8,3 +8,60 @@ $(MREF std, regex) and $(MREF std, algorithm). They will be removed from
 $(MREF std, string) on May 2018.
 
 If you still need to use these, please see $(LINK2 https://github.com/dlang/undeaD, undeaD).
+
+The following are examples of the deprecated functions, and their modern replacements.
+Note that the new code is based on the range model, and therefore does not have
+mutate against the original string, unlike the old code.
```

Comment:
This is just wrong. `removechars`, `squeeze` and `munch` all return a string; only `munch` also takes the argument by value and mutates it. `inPattern` doesn't return/mutate a string at all.

I think it's better to remove this node and illustrate the example with dummy variables.